tests/integrationv2/providers.py

```diff
@@ -156,10 +170,12 @@ def setup_client(self):
         if self.options.reconnect is True:
             cmd_line.append('-r')
 
-        cipher_prefs = 'test_all_tls12'
+        # If the test provided a cipher (security policy) that is compatible with
+        # s2n, we'll use it. Otherwise, default to the appropriate `test_all` policy.
+        cipher_prefs = TEST_ALL_TLS12
         if self.options.protocol is Protocols.TLS13:
-            cipher_prefs = 'test_all'
-        if self.options.cipher in TLS12_PQ_CIPHER_PREFS:
+            cipher_prefs = TEST_ALL
+        if (self.options.cipher is not None) and (self.options.cipher.s2n is True):
```

Comment:
I think you can just write "self.options.cipher and self.options.cipher.s2n". None and False are both false in python.